notebooks/official/custom/SDK_Custom_Container_Prediction.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Copyright 2022 Google LLC
</pre><p>Don't update this. It should reflect when the notebook was created, not updated.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/503/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
noted ! reverted back to 2021.